### DISCUSSION TITLE
Lost trying to get wifi clients to access internet via usb-modem

### DISCUSSION THREAD

**Discussion by raphael303** [RaspAP/raspap-webgui#1604] [General]
RaspberryPi 4, bookworm 64bit lite.
Wlan0 connected to my home wifi
eth0, not connected
Wlan1 usb-wlan stick creating a wifi.
usb0 huawei e3372 modem

The modem light is green and the "networking" panel looks like so, if I access raspap from my homewifi:
<img width="1274" alt="image" src="https://github.com/RaspAP/raspap-webgui/assets/60788818/e0c6fcc7-5429-4868-853d-2281a0c5e993">

and like so, if I access it from raspap wifi:
<img width="994" alt="image" src="https://github.com/RaspAP/raspap-webgui/assets/60788818/24932f52-8461-4c9b-aae4-3a819d031f28">

DHCP Server settings like so:
<img width="586" alt="image" src="https://github.com/RaspAP/raspap-webgui/assets/60788818/20982eb0-23da-41b2-bd0f-f826713c96d9">

and
<img width="497" alt="image" src="https://github.com/RaspAP/raspap-webgui/assets/60788818/70a87a6d-c58a-48bc-aa3b-fc3d0f4c109a">

If I enable "Install a default route for this interface" for usb0, after reboot dnsmasq is down and wont start. clients on the raspap wifi don't get an IP anymore.

No matter what I do, so far I can't get anything to connect to the internet through raspap.
My wish would be that raspap uses wifi (via wlan0) to connect to the internet, and if it is not available uses the usb modem as a fallback. Best without my interference.

Is this possible? And why don't I have any internet at all so far?
Thank you for your time.

**Comment by raphael303** [comment]
ok. I believe this has something to do with homeassistant running on the same device and dbus and systemd-resolved and what not interfering with each other... probably a rabbit hole...

After all my docker attempts have failed I believe it is just impossible to get home assistant and raspap running on the same system...

**Reply by billz** [reply]
This was discussed in https://github.com/RaspAP/raspap-docker/discussions/40. The cgroups option used by RaspAP docker isn't available in HA. The workaround in this case is to run these on separate devices.
